### PR TITLE
[WeatherApp] create interface, UI

### PR DESCRIPTION
**Pull request by mdobatkin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,10 +10,8 @@
     "@types/node": "^16.11.34",
     "@types/react": "^18.0.9",
     "@types/react-dom": "^18.0.3",
-    "@typescript-eslint/eslint-plugin": "^5.23.0",
-    "@typescript-eslint/parser": "^5.23.0",
     "axios": "^0.27.2",
-    "eslint": "^8.15.0",
+
```
Comment (IslamRustamov1) [inline package.json:14]: no empty lines

File: src/Component/preloader/preloader.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
-import React from 'react';
-import {Col, Preloader, Row} from 'react-materialize';
+
```
Comment (IslamRustamov1) [inline src/Component/preloader/preloader.tsx]: no empty lines

File: src/Containers/WeatherToday/components/searchBar/searchBar.tsx [hunk 3]
```diff
@@ -1,19 +1,17 @@
 import React from 'react';
 import {TextInput} from 'react-materialize';
 import './css/style.css';
-
 type SearchBarType = {
-    filterCity: (e: string) => void
+    doCityFilter: (e: string) => void
```
Comment (IslamRustamov1) [inline src/Containers/WeatherToday/components/searchBar/searchBar.tsx]: this was supposed to be filterCity

File: src/Containers/WeatherToday/weatherToday.tsx [hunk 4]
```diff
@@ -1,76 +1,132 @@
...
+        cityTitle: '',
+        humidity: '',
+        feelsLike: 0,
+        temperature: 0,
+        pressure: 0,
+})
     const [isLoading, setIsLoading] = useState<boolean>(false)
-    const [weather, setWeather] = useState<WeatherStateInterface | undefined>()
-    let filterTimeout: NodeJS.Timeout | undefined;
+    const apiKEY = '362efd6a69303a0f5662a19ce25e5e89'
```
Comment (IslamRustamov1) [inline src/Containers/WeatherToday/weatherToday.tsx]: .env

File: src/Containers/WeatherToday/weatherToday.tsx [hunk 5]
```diff
@@ -1,132 +1,71 @@
...
-        </>
+         <div>
+             <SearchBar
+                 filterCity={filterCity}
+             />
+             {weather ?
+                 <WeatherCard
+                 weather={weather}
+             />
+             : <div></div>}
```
Comment (IslamRustam) [inline src/Containers/WeatherToday/weatherToday.tsx]:
```
 <SearchBar filterCity={filterCity} />
 {!!weather && <WeatherCard weather={weather} />}
```

**Pull request merged**